website/source/docs/modules/sources.html.markdown

```diff
@@ -107,6 +107,25 @@ module "consul" {
 
 Bitbucket URLs will require that Git or Mercurial is installed on your system, depending on the type of repository.
 
+## Private Bitbucket Repos
+Private bitbucket repositories must be specified similar to the Generic Git Respository section below.
+
+```hcl
+module "consul" {
+  source = "git::https://bitbucket.org/foocompany/module_name.git
+}
+```
+
+You can also specify branches and version withs the ?ref query
+
+```hcl
+module "consul" {
+  source = "git::https://bitbucket.org/foocompany/module_name.git?hotfix
+}
+```
+
+You will need to run a `terraform get` -update=true if you want to pull the latest versions. This can be handy when you are rapidly iterating on a module in development.
```

Comment:
@sprutner feel free to make the change, or I can. Want to give full credit where it's due 😄